ai/src/ai/common/entity_store.py

```diff
@@ -0,0 +1,53 @@
+import os
+from typing import Generic, TypeVar
+
+from pydantic import BaseModel
+
+T = TypeVar("T", bound=BaseModel)
+
+
+def get_data_path(dirname: str) -> str:
+  return os.path.join(
+    os.path.dirname(__file__), "..", "..", "..", "data", dirname
+  )
+
+
+class EntityStore(Generic[T]):
+  def __init__(self, entity_type: type[T], *, dirname: str):
```

Comment:
I'm not sure, I like dealing with files because then you can version control it with something like Git, but I could also imagine using a document DB like FireStore and storing it there. Basically, if the pain around collaborating with Git is too high, we can switch to a proper DB, but I don't think it'll be too bad because (in theory) the stores should abstract all FS/DB access.